### DISCUSSION TITLE
Does it work for spanish? Spanish from Spain, not from Latinamerica.

### DISCUSSION THREAD

**Discussion by venturaEffect** [voicepaw/so-vits-svc-fork#1032] [Q&A] [answered]
### Is your feature request related to a problem? Please describe.

It is a cool tool but I would like to know if it works for spanish

### Describe alternatives you've considered

It would be great if it is still not possible to make it also work in spanish.

### Additional context

If it isn't possible. Do you know another alternative? Appreciate

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Are you willing to resolve this issue by submitting a Pull Request?

- [X] Yes, I have the time, and I know how to start.
- [ ] Yes, I have the time, but I don't know how to start. I would need guidance.
- [ ] No, I don't have the time, although I believe I could do it if I had the time...
- [ ] No, I don't have the time and I wouldn't even know how to start.

**Comment by 34j** [comment, marked as answer]
Basically yes, but training with Spanish dataset may improve pronunciation.
